### ISSUE TITLE
Error "oops" with 242fe52375ec27bedaa6bd67de20c61203d148bf

### ISSUE THREAD

**Issue by zickgraf** [gap-packages/AutoDoc#246]
I just tried to test #245 but get an error on the master branch (bisected to 242fe52375ec27bedaa6bd67de20c61203d148bf) with https://github.com/zickgraf/FinSetsForCAP/:

```
Extracting manual examples for FinSetsForCAP package ...
43 singles detected
Single 1 : Error, oops at /.../pkg/AutoDoc/gap/AutoDocMainFunction.gi:486 called from
(AUTODOC_ExtractMyManualExamples)(
    pkgname,
    pkgdir,
    doc_dir,
    gapdoc.main,
    gapdoc.files,
    extract_examples
); at /.../pkg/AutoDoc/gap/Magic.gi:651 called from
<function "AutoDoc">( <arguments> )
 called from read-eval loop at makedoc.g:27
```

If you cannot reproduce easily, I can look into this on Monday.

Edit: Looking at the commit it might be a local problem with unusual installation paths or something. Also I have not tried to use a clean copy of FinSetsForCAP. I will provide more details on Monday :-)

**Comment by fingolfin** [comment]
I can reproduce it. This is caused by my `_Chunks.xml` fix, but it really is a bug in `AUTODOC_ExtractMyManualExamples` which would have broken with any other absolute path as well. Specifically, in the break loop:
```
brk> location;
"/Users/mhorn/Projekte/GAP/repos/pkg/homalg_more/FinSetsForCAP/doc/_Chunks.xml"
```
Should be fixed by PR #247

**Comment by zickgraf** [comment]
Yes, with #247 this error does not occur anymore. Thanks for the quick fix!
